static/js/pages/SitesDashboard.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from "react"
+import { useSelector } from "react-redux"
+import { useMutation, useRequest } from "redux-query-react"
+import { RouteComponentProps } from "react-router-dom"
+import { FormikHelpers } from "formik"
+import { isString } from "lodash"
+
+import { SiteForm, SiteFormValues } from "../components/forms/SiteForm"
+
+import {
+  getTransformedWebsiteName,
+  websiteMutation,
+  websiteStartersRequest
+} from "../query-configs/websites"
+import { startersSelector } from "../selectors/websites"
+import { getResponseBodyErrors, isErrorResponse } from "../lib/util"
+import { siteUrl } from "../lib/urls"
+
+type Props = RouteComponentProps<Record<string, never>>
+
+export default function SitesDashboard({ history }: Props): JSX.Element | null {
+  const [starterQueryState] = useRequest(websiteStartersRequest())
+  const [createWebsiteQueryState, createWebsite] = useMutation(payload =>
+    websiteMutation(payload)
+  )
+
+  const onSubmitForm = async (
+    values: SiteFormValues,
+    { setErrors, setSubmitting, setStatus }: FormikHelpers<SiteFormValues>
+  ) => {
+    if (!values.starter || createWebsiteQueryState.isPending) {
+      return
+    }
+    const response = await createWebsite({
+      // @ts-ignore
+      title:      values.title,
+      // @ts-ignore
+      starter_id: values.starter
+    })
+    if (!response) {
+      return
+    }
+    // Display errors if any were returned
+    if (isErrorResponse(response)) {
+      const errors = getResponseBodyErrors(response)
+      if (!errors) {
+        return
+      } else if (isString(errors)) {
```

Comment:
FWIW, I got some feedback on several PRs awhile back that we should try to avoid lodash and use ramda if at all possible.  `R.is(String, errors)` should probably work.  Though personally I find lodash much easier to understand 😁

Reply:
If nothing else it will be good to cut down on the number of libraries